### DISCUSSION TITLE
Customize account credit expiring soon notification days threshold

### DISCUSSION THREAD

**Discussion by redactedscribe** [mullvad/mullvadvpn-app#9087] [Feature requests & Ideas]
App version: 2022.1

### Issue description
Some payment methods take longer to process than others, cash taking the longest I believe. It would be useful to be able to customize the number of days before the account credit expires that the notification should start appearing. If not customizable by the user, then maybe have different hard-coded values based upon the last payment method used (if this is known)? E.g. If cash was used, start showing the credit expiring soon notification 14 days in advance to factor in payment sending and processing time.

Preferably the value would be settable by the user as some people with larger amounts of credit (multiple months) may want more than the current number of days worth of notifications, compared to those with only a month of credit. From my experience, the notification only appears about 3 days prior to the credit expiring, maybe 5?

Thanks.

**Comment by faern** [comment]
The app has no way of knowing what payment methods are used. This is not visible on the website neither for that matter. This is by design. Stealing someone else's Mullvad account number should not disclose anything about them (such as payment methods they use).

You are correct, I don't think we can raise the hardcoded default by a lot. People who renew monthly and use a fast payment method should not be bothered with this notification a large part of the month.

I'd say this is only a problem for users paying with cash. And given that it complicates the UI and the app (Every setting makes the app harder to understand for the user) I'm not sure if this is worth implementing. But I'll take it with the team and we'll see.
